src/kvstore.rs

```diff
@@ -1,9 +1,11 @@
 use chashmap::CHashMap;
-use chrono::{DateTime, Utc};
+
+//use chrono::{DateTime, Utc};
 
 pub struct KvElement {
     data: Vec<u8>,
-    created: DateTime<Utc>,
+//    created: DateTime<Utc>,
+//    locked: bool
```

Comment:
When this is implemented, you could add:

 - `last_edit`
 - `mime_type`
 - `created_author`
 - `last_edit_author`

The last two ones would need to add an unique identifier (uuid) in the JWT to be injected here (to know which JWT interracted with this kv).

Reply:
For last_edit and mime_type I agree, but for author I'm not sure, but this `fields` are commented because they need to be implemented before, I will complete the metadata when it will be the time.